pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -110,13 +110,25 @@ class Build {
 
     def runTests() {
         def testStages = [:]
-        List testList = []
-
+        List testList = buildConfig.TEST_LIST
+        def java_to_build = buildConfig.JAVA_TO_BUILD
+        jdk_branch = 'dev'
+        suffix="adoptopenjdk/openjdk-${java_to_build}"
```

Comment:
could suffix be declared as a def or static type?

Reply:
Updated.